tg_bot/__main__.py

```diff
@@ -415,6 +415,37 @@ def migrate_chats(bot: Bot, update: Update):
     LOGGER.info("Successfully migrated!")
     raise DispatcherHandlerStop
 
+def is_chat_allowed(bot, update):
+    if len(WHITELIST_CHATS) != 0:
+        chat_id = update.effective_message.chat_id
```

Comment:
Extract this from the if statements. You use it in every `if` block, redeclaring it is unecessary.